werkzeug/wrappers.py

```diff
@@ -1434,6 +1458,23 @@ def make_conditional(self, request_or_environ):
         :param request_or_environ: a request object or WSGI environment to be
                                    used to make the response conditional
                                    against.
+        :param accept_ranges: This parameter dictates the value of
+                              `Accept-Ranges` header. If ``False`` (default),
+                              the header is not set. If ``True``, it will be set
+                              to ``"bytes"``. If ``None``, it will be set to
+                              ``"none"``. If it's a string, it will use this
+                              value.
+        :param complete_length: Will be used only in valid Range Requests.
+                                It will set `Content-Range` complete length
+                                value and compute `Content-Length` real value.
+                                This parameter is mandatory for successful
+                                Range Requests completion.
+        :param ignore_invalid_range_requests: set to ``True`` to silently degrade
+                                              to complete HTTP response for
+                                              invalid range requests, instead of
+                                              raising an HTTP 406 exception.
```

Comment:
Should be "416" instead of "406", also the spec says:

>An origin server MUST ignore a Range header field that contains a range unit it does not understand.

and:

>  A server that supports range requests MAY ignore or reject a Range header field that consists of more than two overlapping ranges, or a set of many small ranges that are not listed in ascending order,

but also:

>If all of the preconditions are true, the server supports the Range header field for the target resource, and the specified range(s) are invalid or unsatisfiable, the server SHOULD send a 416 (Range Not Satisfiable) response.

So I think more nuance is necessary for errorhandling. I don't think we need a user-visible option for this though, unless there is a practical argument for violating the spec.